pysoundcomparisons/__main__.py

```diff
@@ -511,106 +502,79 @@ def write_modified_soundfiles(args):
                 if len(lineArray) > 0:
                     valid_soundfilepaths.append(lineArray[-1])
                 line = fp.readline().strip()
-
-    catalog_filepath = api.repos.joinpath('soundfiles', 'catalog.json')
-    if catalog_filepath.exists():
-        catalog_items = jsonlib.load(catalog_filepath)
     else:
-        print("File path {} does not exist.".format(catalog_filepath))
-        return return_data
+        print("'valid_soundfilepaths.txt' can not be found at %s" % (
+            valid_soundfilepaths_filepath))
+        return
+
+    catalog = MediaCatalog(api.repos.joinpath('soundfiles', 'catalog.json'))
+    try:
+        catalog[0]
+    except:
+        print("catalog.json at {} is empty.".format(
+            api.repos.joinpath('soundfiles', 'catalog.json')))
```

Comment:
This catalog initialization block may be a candidate for a small utility function in `__main__` - at least it shows up here for the second time. Maybe this should raise [`clldutils.clilib.ParserError`](https://github.com/clld/clldutils/blob/master/src/clldutils/clilib.py#L12), because the typical reason for this error to occur may be an incorrect repository location?